testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/opentracing/SubsystemConfigurationTestCase.java

```diff
@@ -59,6 +65,14 @@
 @ServerSetup(SubsystemConfigurationTestCase.SetupTask.class)
 public class SubsystemConfigurationTestCase {
 
+    private static final String OUTBOUND_BINDING = "jaeger-sender";
+    private static final ModelNode OUTBOUND_BINDING_ADDRESS = PathAddress
+            .pathAddress(SOCKET_BINDING_GROUP, "standard-sockets")
+            .append(REMOTE_DESTINATION_OUTBOUND_SOCKET_BINDING, OUTBOUND_BINDING)
+            .toModelNode();
+    private static final int SENDER_PORT = 6832;
+    private static final String SENDER_HOST = NetworkUtils.formatPossibleIpv6Address(System.getProperty("node0", "localhost"));
```

Comment:
This should be using `TestSuiteEnvironment.getServerAddress()`. That method already formats ipv6 addresses and also checks the two properties `management.address` and `node0`.